pkg/checksum/checksum.go

```diff
@@ -5,37 +5,20 @@ import (
 	"encoding/base64"
 	"encoding/json"
 	"fmt"
-	"io"
-	"net/http"
 
 	"github.com/Azure/azure-sdk-for-go/services/resources/mgmt/2019-05-01/resources"
 	"github.com/giantswarm/microerror"
 )
 
 func GetDeploymentTemplateChecksum(deployment resources.Deployment) (string, error) {
-	templateLink := deployment.Properties.TemplateLink
-	if templateLink == nil {
-		return "", microerror.Mask(nilTemplateLinkError)
-	}
-
-	resp, err := http.Get(*templateLink.URI)
+	template := deployment.Properties.Template.(map[string]interface{})
```

Comment:
Instead of downloading the template link, we calculate the checksum from the template property of the `Deployment`.